semantic-conventions/src/opentelemetry/semconv/model/semantic_attribute.py

```diff
@@ -84,7 +87,7 @@ def is_enum(self):
 
     @staticmethod
     def parse(
-        prefix, semconv_stability, yaml_attributes
+        prefix, semconv_stability, yaml_attributes, parse_template_attributes=False
```

Comment:
I think we should rather not mix filtering and parsing. If we want to filter them out, we can do so after parsing

Reply:
agree